docs/feathub_expression.md

```diff
@@ -34,28 +34,30 @@ Arithmetic functions take numeric values as inputs and outputs a numeric value.
 
 Comparison functions take numeric values as inputs and outputs a boolean value.
 
-| Function | Description |
-| --- | --- |
-| x > y | Returns TRUE iff `x > y`. Returns NULL if in case of error. |
-| x >= y | Returns TRUE iff `x >= y`. Returns NULL if in case of error. |
-| x < y | Returns TRUE iff `x < y`. Returns NULL if in case of error. |
-| x <= y | Returns TRUE iff `x <= y`. Returns NULL if in case of error. |
-| x = y | Returns TRUE iff `x` is equal to `y`. Returns NULL if in case of error. |
-| x <> y | Returns TRUE iff `x` is not equal to `y`. Returns NULL if in case of error. |
+| Function      | Description                                                              |
+|---------------|--------------------------------------------------------------------------|
+| x > y         | Returns TRUE iff `x > y`. Returns NULL in case of error.                 |
+| x >= y        | Returns TRUE iff `x >= y`. Returns NULL in case of error.                |
+| x < y         | Returns TRUE iff `x < y`. Returns NULL in case of error.                 |
+| x <= y        | Returns TRUE iff `x <= y`. Returns NULL in case of error.                |
+| x = y         | Returns TRUE iff `x` is equal to `y`. Returns NULL in case of error.     |
+| x <> y        | Returns TRUE iff `x` is not equal to `y`. Returns NULL in case of error. |
+| x IS NULL     | Returns TRUE iff `x` is NULL.                                            |
+| x IS NOT NULL | Returns TRUE iff `x` is not NULL.                                        |
 
 ## Logical Functions
 
-| Function        | Description                                     |
-|-----------------|-------------------------------------------------|
-| bool1 OR bool2  | Returns TRUE if bool1 is TRUE or bool2 is TRUE. |
-| bool1 AND bool2 | Returns TRUE if bool1 and bool2 are both TRUE.  |
+| Function                 | Description                                     |
+|--------------------------|-------------------------------------------------|
+| bool1 &#124;&#124; bool2 | Returns TRUE if bool1 is TRUE or bool2 is TRUE. |
```

Comment:
`&#124;` stands for `|`. Given that "|" is a syntax in markdown table, we cannot directly write this character in the document[1]. This problem is not applicable now as this PR changes to `AND/OR` instead of `&&/||`.

[1] https://stackoverflow.com/questions/17319940/how-to-escape-a-pipe-char-in-a-code-statement-in-a-markdown-table